asterisk_click2dial/asterisk_click2dial.py

```diff
@@ -147,49 +291,74 @@ def _reformat_number(self, cr, uid, erp_number, ast_server, context=None):
         if country_prefix == tmp_number[0:len(country_prefix)]:
             # If the number is a national number,
             # remove 'my country prefix' and add 'national prefix'
-            tmp_number = (national_prefix) + tmp_number[len(country_prefix):len(tmp_number)]
-            _logger.debug('National prefix = %s - Number with national prefix = %s' % (national_prefix, tmp_number))
+            tmp_number = (national_prefix
+                          + tmp_number[len(country_prefix):len(tmp_number)])
+            _logger.debug(
+                'National prefix = %s - Number with national prefix = %s'
+                % (national_prefix, tmp_number)
```

Comment:
don't use '%' in calls to logger methods, pass the substitution values as additional positional arguments.